.changeset/65-update-project-to-be-plugin.md

```diff
@@ -0,0 +1,10 @@
+---
+issue: https://github.com/pmedianetwork/eslint-config/issues/65
+type: major
+---
+
+Changed structure of project from config, to a plugin.
```

Comment:
can we add a migration path here?

Reply:
👍 
https://github.com/pmedianetwork/eslint-config/pull/64/commits/928c075102879ea90f0b0c1b491aa7f0ba0a11c8